.travis.yml

```diff
@@ -1,16 +1,15 @@
-language: go
 sudo: false
+language: go
 
 go:
+  - 1.6.x
   - 1.7.x
 
 services:
   - redis-server
   - mongodb
 
 install:
-  - go get -d -v ./...
-  - go get -d -v -tags coprocess ./...
```

Comment:
(also explained in the commit, let me know if you'd like my commits to be more verbose)